lib/rules/order-in-models.js

```diff
@@ -1,55 +1,57 @@
 'use strict';
 
-var utils = require('../utils/utils');
 var ember = require('../utils/ember');
+var reportUnorderedProperties = require('../utils/property-order').reportUnorderedProperties;
+
+const ORDER = [
+  'attribute',
+  'relationship',
+  'single-line-function',
+  'multi-line-function',
+  'other',
+];
+
+const NAMES = {
+  'attribute': 'attribute',
+  'relationship': 'relationship',
+  'single-line-function': 'single-line function',
+  'multi-line-function': 'multi-line function',
+  'other': 'property',
```

Comment:
the problem is that "unknown" is different than "other". For "unknown" we really don't know what type of property it could be because we should've caught all the ones we know about. For "other" we don't analyze all and treat all the ones we haven't analyzed further the same.